eth/vm/computation.py

```diff
@@ -113,8 +113,8 @@ class BaseComputation(Configurable, ABC):
     return_data = b''
     _error = None  # type: VMError
 
-    _log_entries = None  # type: List[Tuple[int, bytes, List[int], bytes]]
-    accounts_to_delete = None  # type: Dict[bytes, bytes]
+    _log_entries = None  # type: List[Tuple[int, Address, List[int], bytes]]
+    accounts_to_delete = None  # type: Dict[Address, Address]
```

Comment:
Why did you have to change this? My guess is that the previous type was already wrong but somehow mypy didn't notice until you made this PR?

Reply:
> Why did you have to change this?

mypy started complaining

> My guess is that the previous type was already wrong but somehow mypy didn't notice until you made this PR?

Yup, it can be hard to tell which things are our fault for misconfiguration, and which things are mypy bugs. So I tend not to dig in until it complains in a way that makes sense to me. I also read it as "mypy didn't notice before"